### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by OsamaAbbas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,6 +125,10 @@ curl.request({ url: 'http://google.com', pretend: true }, function (err, stdout,
 });
 ```
 
+`curl_path` - *default: 'curl'*
+
+Use this if you want to use curlrequest in Windows or if you want to specify anothr path for curl.
```
Comment (chriso) [inline README.md]: You've got a typo here: `anothr`. I think it's sufficient to say: *Use this to specify an alternative path for curl.*

**Comment by OsamaAbbas** [comment]
Ok. I guess you should edit it directly in your base branch. Thank you :) .

**Comment by OsamaAbbas** [comment]
It is fixed now as you suggested.

**Comment by chriso** [comment]
Awesome, thanks.

**Pull request merged**